examples/default/src/App.tsx

```diff
@@ -22,10 +17,16 @@ export const App: React.FC = () => {
       invocationEvents: [InvocationEvent.floatingButton],
       debugLogsLevel: LogLevel.verbose,
     });
-    CrashReporting.setNDKCrashesEnabled(true);
-
     Instabug.setReproStepsConfig({
-      all: ReproStepsMode.enabled,
+      bug: ReproStepsMode.enabledWithNoScreenshots,
+      allCrashes: ReproStepsMode.enabledWithNoScreenshots,
+      anr: ReproStepsMode.enabledWithNoScreenshots,
+      appHangs: ReproStepsMode.enabledWithNoScreenshots,
+      sessionReplay: ReproStepsMode.enabledWithNoScreenshots,
+      fatalCrash: ReproStepsMode.enabledWithNoScreenshots,
+      nonFatalCrash: ReproStepsMode.enabledWithNoScreenshots,
+      forceRestart: ReproStepsMode.enabledWithNoScreenshots,
+      outOfMemory: ReproStepsMode.enabled,
```

Comment:
We have some options overriding others here which can cause some confusion, let's probably keep all enabled as it was or use a config with no conflicting options if needed.